src/EnergyPlus/WeatherManager.cc

```diff
@@ -4507,14 +4057,13 @@ namespace WeatherManager {
                                       Real64 const EqOfTime,     // Equation of Time
                                       Real64 const SinSolDeclin, // Sine of Solar Declination
                                       Real64 const CosSolDeclin, // Cosine of Solar Declination
-                                      Array1D<Real64> &SUNCOS)
+                                      Vector3<Real64> &SUNCOS)
```

Comment:
`{x,y,z}` coordinates should be `Vector3` (which is a struct) not `Array1D` (which is a heap vector).